packages/costa/src/runnable.ts

```diff
@@ -193,9 +193,14 @@ export class PluginRunnable {
    * @param op
    * @param msg
    */
-  private send(op: PluginOperator, msg?: PluginMessage): boolean {
+  private send(op: PluginOperator, msg?: PluginMessage): Promise<void> {
```

Comment:
```suggestion
  private async send(op: PluginOperator, msg?: PluginMessage): Promise<void> {
```

Reply:
Not necessary, no `await` used in thin function.